src/integrations/ga/GaToSplit.js

```diff
@@ -1,7 +1,257 @@
-// @TODO implement real module
-// eslint-disable-next-line no-unused-vars
-function GaToSplitFactory(options, storage, coreSettings) {
-  return {};
+import { isString, isNumber, uniqO } from '../../utils/lang';
+import logFactory from '../../utils/logger';
+import {
+  validateEvent,
+  validateEventValue,
+  validateEventProperties,
+  validateKey,
+  validateTrafficType,
+} from '../../utils/inputValidation';
+const log = logFactory('splitio-ga-to-split');
+
+/**
+ * Provides a plugin to use with analytics.js, accounting for the possibility
+ * that the global command queue has been renamed or not yet defined.
+ * @param {string} pluginName The plugin name identifier.
+ * @param {Function} pluginConstructor The plugin constructor function.
+ */
+function providePlugin(pluginName, pluginConstructor) {
+  const gaAlias = window.GoogleAnalyticsObject || 'ga';
+  window[gaAlias] = window[gaAlias] || function (...args) {
+    (window[gaAlias].q = window[gaAlias].q || []).push(args);
+  };
+
+  // provides the plugin for use with analytics.js.
+  window[gaAlias]('provide', pluginName, pluginConstructor);
+
+  // @TODO should we registers the plugin on the global gaplugins object?
+  // window.gaplugins = window.gaplugins || {};
+  // window.gaplugins[capitalize(pluginName)] = pluginConstructor;
+}
+
+// Default filter: accepts all hits
+export function defaultFilter() { return true; }
+
+// Default mapping: object used for building the default mapper from hits to Split events
+// @TODO review default mapping.
+const defaultMapping = {
+  eventTypeIdPrefix: {
```

Comment:
I don't know if prefix is the right name for these.

Reply:
Removed to simplify default mapper and since a prefix can be passed as options ('ga' by default)